apps/src/netsim/NetSimStatusPanel.js

```diff
@@ -77,11 +73,9 @@ NetSimStatusPanel.prototype.render = function (data) {
   this.getBody().html(newMarkup);
 
   // Add a button to the panel header
-  if (data.isConnected) {
-    this.addButton(
-        i18n.disconnectButton({ caret: '<i class="fa fa-caret-left"></i>' }),
-        this.disconnectCallback_);
-  }
+  this.addButton(
+      i18n.disconnectButton({ caret: '<i class="fa fa-caret-left"></i>' }),
```

Comment:
I'm told not to worry about internationalization for now but I've tried to do it right most places to minimize pain later.